app/actions/ats_client.py

```diff
@@ -64,6 +64,17 @@ def __init__(self, error: Exception, message: str, status_code=422):
         super().__init__(f"'{self.status_code}: {self.message}, Error: {self.error}'")
 
 
+def normalize_xml_string(xml_str):
+    # Remove surrounding quotes if present
+    if xml_str.startswith('"') and xml_str.endswith('"'):
+        xml_str = xml_str[1:-1]
```

Comment:
Here you might want to use `strip()` instead.

```suggestion
    xml_str = xml_str.strip('"')
```